CHANGELOG.md

```diff
@@ -92,6 +92,13 @@
 - Bumped lua-resty-session from 4.0.2 to 4.0.3
   [#10338](https://github.com/Kong/kong/pull/10338)
 
+### Additions
```

Comment:
We already have a `Additions` section, please move the entry to the correct place.